Src/BootCamp.Chapter/Computer/Body.cs

```diff
@@ -1,10 +1,12 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace BootCamp.Chapter.Computer
+﻿namespace BootCamp.Chapter.Computer
 {
     public class Body
     {
+        private readonly string _body;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_body' or refactor the code to use its value.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5124353)